mitmproxy/optmanager.py

```diff
@@ -416,6 +416,20 @@ def dump_defaults(opts):
     return ruamel.yaml.round_trip_dump(s)
 
 
+def dump_dicts(opts):
+    """
+        Dumps the options into a list of dict object.
+
+        Return: A list like: [ { name: "anticahce", type: "bool", default: false, value: true, help: "help text"}]
+    """
+    options_list = []
+    for k in sorted(opts.keys()):
+        o = opts._options[k]
```

Comment:
I'm afraid we could not do this, `opts` here is not a `dict`, but a `OptManager`, or do you think we should add a `.items` method for it?